dockerscripts/docker-entrypoint.sh

```diff
@@ -37,7 +37,67 @@ docker_secrets_env() {
     fi
 }
 
+
+
+docker_setuid_gid() {
+    if  [ -z "$MINIO_GID" ] ||  [ -z "$MINIO_UID" ] ; then
+	    groupadd -g 190 minio && useradd -u 190 -g minio minio
+    else
+      groupadd -g $MINIO_GID  minio && useradd -u $MINIO_UID  -g minio minio
+    fi
+}
+
+# switch to non root (minio) user if there isn't an existing .minio.szys directory or
+# if its an empty data directory. We dont switch to minio user if there is an existing
+# data directory, this is because we'll need to chown all the files -- which may take
+# too long, ruining the user's experience.
```

Comment:
we can better word this - `ruining the user's experience` is not a good comment here.